FusionIIIT/applications/visitor_hostel/migrations/0002_auto_20230401_1704.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 3.1.5 on 2023-04-01 16:04
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('visitor_hostel', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='bill',
+            name='bill_date',
+            field=models.DateField(blank=True, default=django.utils.timezone.now),
+        ),
+        migrations.DeleteModel(
+            name='MealRecord',
+        ),
+        migrations.CreateModel(
+            name='MealRecord',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('meal_date', models.DateField()),
+                ('morning_tea', models.IntegerField(default=0)),
+                ('eve_tea', models.IntegerField(default=0)),
+                ('breakfast', models.IntegerField(default=0)),
+                ('lunch', models.IntegerField(default=0)),
+                ('dinner', models.IntegerField(default=0)),
+                ('persons', models.IntegerField(default=0)),
+                ('booking', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='visitor_hostel.bookingdetail')),
+                ('visitor', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='visitor_hostel.visitordetail')),
+            ],
+        ),
+        migrations.DeleteModel(
+            name='Bill',
+        ),
+        migrations.CreateModel(
+            name='Bill',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('meal_bill', models.IntegerField(default=0)),
+                ('room_bill', models.IntegerField(default=0)),
+                ('payment_status', models.BooleanField(default=False)),
+                ('booking', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, to='visitor_hostel.bookingdetail')),
+                ('caretaker', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
+                ('room', models.ManyToManyField(to='visitor_hostel.RoomDetail')),
+                ('bill_date', models.DateField(default=django.utils.timezone.now, blank=True))
+            ],
+        ),
+    ]
```

Comment:
Please don't do it this way, you are deleting a model and later adding a new field. Just adding a new field or altering it will automatically create a migration.